src/components/generators/uischema/uischema-service.ts

```diff
@@ -0,0 +1,5 @@
+import {UISchemaGenerator} from "./jsonforms-uischemagenerator";
```

Comment:
why not register in jsonforms-uischemagenerator, also see jsonforms-schemagenerator